src/modules/Comments.js

```diff
@@ -0,0 +1,46 @@
+let appID = 'WXXsCPtk675NtAM8NnCG';
+const apiInvolvement = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/';
+
+// temporary code - get app id
+// fetch(apiInvolvement, { method: 'POST' })
+//   .then((result) => result.text())
+//   .then((parsed) => { appID = parsed; })
+//   .then(() => console.log(appID));
+// temporary code ends
+
+export function commentFetch(coin) {
+  fetch(`${apiInvolvement + appID}/comments?item_id=${coin}`)
+    .then((response) => response.json())
```

Comment:
you can use a request instance to fetch data from the api "to separate tasks"

Reply:
I think fetch api reduces the codebase and makes it easier to read and adjust.
Seperating tasks is a valid concern. So I did seperate fetch and popluate tasks.